lib/services.cf

```diff
@@ -218,6 +219,18 @@ bundle agent standard_services(service,state)
       "$(init) $(state)"
       contain => silent;
 
+    sysvservice.freebsd.start.!enabled::
+      "/usr/sbin/sysrc $(service)_enable=YES"
+      handle => "standard_services_sysvservice_enable",
+      classes => kept_successful_command,
```

Comment:
Why do you want to override the promsie outcome here?  If the command returns zero it will be considered a repair, but with this kept_successful_command classes body, return zero will be interpreted as a promise kept. Since the system is modified I think repair makes sense, but maybe I am missing something.

Reply:
I agree that doesn't make sense.